toolkit/templates/forms/_selection-button.html

```diff
@@ -20,6 +20,11 @@
         {{ option.description }}
       </p>
     {% endif %}
+    {% if option.hint %}
+      <span class="form-hint">
```

Comment:
FWIW this looks wrong to me - I think it should have been a `div`? (it's at the same level as a `p` which is block-level). Sorry I realise I'm too late and you'd have to refactor the tests on your other PR :(

Reply:
Hey @galund , 

I personally think block-level elements should not sit inside labels. It should always be a `span`.

I found this on the internet which seems to back up what I'm saying:
https://stackoverflow.com/a/4461959

The `p` tag above should probably change to a `span`

Reply:
Oh yeah, see what you mean. 👍